com.unity.netcode.gameobjects/Runtime/Core/SnapshotSystem.cs

```diff
@@ -240,6 +246,11 @@ internal void AddSpawn(SnapshotSpawnCommand command)
                     Spawns[NumSpawns] = command;
                     NumSpawns++;
                 }
+
+                foreach (var dstClientId in command.TargetClientIds)
+                {
+                    m_NetworkManager.NetworkMetrics.TrackObjectSpawnSent(dstClientId, command.NetworkObject, 8);
```

Comment:
It's unfortunate though maybe negligible that we 1) have to call 'n' times for n objects, and 2) still call 'n' times even when metrics are turned off.

Reply:
yeah. Probably something to revisit. But in a perfect world, we'd run a sampling profiler and address the hotspots first :-). However inelegant n useless function calls might be, I doubt it's a hotspot.